### DISCUSSION TITLE
How to set clipboard in slint app?

### DISCUSSION THREAD

**Discussion by evan0greenup** [slint-ui/slint#2930] [Q&A]
I have already read https://docs.rs/slint/latest/slint/platform/enum.Clipboard.html

But I have no idea about how to set the string inside clipboard in runtime?

Is this done in slint source code or rust source code?

**Comment by ogoffart** [comment]
Could you elaborate on your usecase?

Anyway, these function are used when writing your own platform and we need callback in slint to implment sopy and paste.
If you're using slint with one of the default ploatform, we don't support that currently
See https://github.com/slint-ui/slint/issues/2570

You will need to use another library to communicate with the clipboard.  For example, the copypasta, or arboard https://docs.rs/arboard/ crate.
